Frontend/src/components/Upload/UploadContainer.jsx

```diff
@@ -21,7 +23,7 @@ function UploadContainer() {
   const [email, setEmail] = useState('');
   const [customerFirstName, setCustomerFirstName] = useState('');
   const [customerLastName, setCustomerLastName] = useState('');
-  const [color, setColor] = useState('');
+  const [color, setColor] = useState('#FFFFFF'); //Set initial color to white
```

Comment:
I think it would be better to change it from white to the first color in the list